src/plugins/controls/title/title.js

```diff
@@ -144,26 +103,32 @@ export default pluginFactory({
                     $timer,
                     hours: currentHours,
                     minutes: currentMinutes,
+                    seconds: currentSeconds,
                     stats,
                     unansweredQuestions: currentUnansweredQuestions,
                 } = this.titles[scope];
                 const time = moment.duration(remainingTime / precision, 'seconds');
                 const hours = time.get('hours');
                 const minutes = time.get('minutes');
+                const seconds = time.get('seconds');
                 const unansweredQuestions = stats && (stats.questions - stats.answered);
 
                 // check if notification should be updated
                 if (
```

Comment:
please remove this `if`, it is not relevant anymore since the `timertick` event triggered every second